extension/content/components/recipes/details/DetailsHeader.js

```diff
@@ -127,6 +134,14 @@ export default function DetailsHeader() {
         {viewExperimentButton}
         {requestApprovalButton}
         {statusToggleButton}
+        <IconButton
+          appearance="primary"
+          className="ml-1"
+          icon={<Icon icon="copy" />}
+          onClick={handleCopyClick}
+        >
+          Copy Recipe
```

Comment:
"Copy" is a bit confusing to me. Especially with that icon it's ambiguous whether you're copying the recipe to the clipboard or "making a copy". I think "Clone" or "Duplicate" are probably better choices for verbiage.